### DISCUSSION TITLE
Dashboard app for WHMCS

### DISCUSSION THREAD

**Discussion by SwimGeek** [chatwoot/chatwoot#10183] [Self hosted]
Hi

Please point me to a sample Dashboard app.

I'm looking to query WHMCS for Client data when a chat starts. Pretty basic query - send email address - get customer details back.

I looked at the way you configure Dashboard apps - seems you only give it a single URL.

I would guess you need to also give it some security credentials - or you need to host the Dashboard app URL on the local server.

Is the idea to send the username and password in the URL via HTTPS?

Do you have advice for using a Dashboard app to do CRM / Client info lookup?

**Comment by i9team** [comment]
Posso criar a solução para whmcs, @SwimGeek me envie uma mensagem no whatsapp: https://wa.me/553597630296
